server/src/core/event/dto/create-event.input.ts

```diff
@@ -0,0 +1,10 @@
+import { ArgsType, Field } from '@nestjs/graphql';
+
+@ArgsType()
+export class CreateEventInput {
+  @Field({ description: 'Type of the event' })
+  type: string;
+
+  @Field({ description: 'Event data in JSON format' })
+  data: string;
```

Comment:
@FelixMalfait GraphQL JSON scalar should be used instead:
https://docs.nestjs.com/graphql/scalars

Reply:
Ah ok nice I thought it wasn't supported!